rosdep/python.yaml

```diff
@@ -1797,6 +1797,55 @@ python-hypothesis:
     yakkety_python3: [python3-hypothesis]
     zesty: [python-hypothesis]
     zesty_python3: [python3-hypothesis]
+python-imageio:
+  alpine:
+    pip:
+      packages: [imageio]
+  arch:
+    pip:
+      packages: [imageio]
+  cygwin:
+    pip:
+      packages: [imageio]
+  debian:
+    '*':
+      pip:
+        packages: [imageio]
+    buster: [python-imageio]
+  fedora:
+    pip:
+      packages: [imageio]
+  freebsd:
+    pip:
+      packages: [imageio]
+  gentoo: [dev-python/imageio]
+  macports:
+    pip:
+      packages: [imageio]
+  opensuse:
+    '*':
+      pip:
+        packages: [imageio]
+    '15.0': [python-imageio]
+  osx:
+    pip:
+      packages: [imageio]
+  rhel:
+    pip:
+      packages: [imageio]
+  slackware:
+    pip:
+      packages: [imageio]
+  ubuntu:
+    '*':
+      pip:
+        packages: [imageio]
+    bionic: [python-imageio]
+    bionic_python3: [python3-imageio]
```

Comment:
I don't believe we're using this system for handling python2 versus python3 keys anymore. There are still a number of instances in the database but instead of adding new ones we should use a separate `python3-LIBRARY` key to resolve the python3 dependency. For packages that wish to support python3 and python2 in the same package, using conditional dependencies which [check the `ROS_PYTHON_VERSION`](http://www.ros.org/reps/rep-0149.html#environment-variables) is the recommended approach.

Same for the cosmic_python3 and disco_python3 entries below.

Reply:
removed them